contracts/deploy/012_upgrades.js

```diff
@@ -108,19 +106,21 @@ const upgrades = async () => {
     log("Upgraded CompoundStrategy to new implementation");
   }
 
-  console.log(`${deployName} deploy done.`);
   return true;
 };
 
-const main = async () => {
+const main = async (hre) => {
   console.log(`Running ${deployName} deployment...`);
-  await upgrades();
+  if (!hre) {
+    hre = require("hardhat");
+  }
+  await upgrades(hre);
   console.log(`${deployName} deploy done.`);
   return true;
 };
 
 main.id = deployName;
 main.dependencies = ["011_ousd_fix"];
-main.skip = () => !isMainnetOrRinkebyOrFork;
+main.skip = () => !(isMainnet || isRinkeby);
```

Comment:
@sparrowDom Remind me, why do we need to skip this deploy on the fork?

Reply:
If it is set to not skip on fork then the deploy is applied immediately when the node in `fork` mode is started. And we are not able to execute the first (before) step of the tests before the contract deploy.

Reply:
ah yeah... that's not ideal since sometime developers will want to run the fork for other reasons than running smoke tests? I can't think of a good solution though. I guess the developer could then update that line... 

I'd suggest we update the contracts/README.md to document all this in the section that covers how to use the fork? (and we should anyway document how to use the smoketests).